src/clarity-icons/shapes/core-shapes.ts

```diff
@@ -425,6 +425,15 @@ const coreShapes: any = {
                 <title>view-columns</title>
                 <path class="clr-i-outline clr-i-outline-path-1" d="M31,5H5A2,2,0,0,0,3,7V29a2,2,0,0,0,2,2H31a2,2,0,0,0,2-2V7A2,2,0,0,0,31,5ZM13,29H5V7h8Zm10,0H15V7h8Z"/>
             </svg>
+        `,
+
+    "angle-double": `
```

Comment:
thanks for letting me know. can you request a change on this so that i dont forget?